src/Augurk.CommandLine/Extensions/SpecFlowEntityExtensions.cs

```diff
@@ -128,146 +154,172 @@ public static Background ConvertToBackground(this SpecFlow.Background background
+        /// <param name="examples">The <see cref="Gherkin.Ast.Examples"/> instance that should be converted.</param>
         /// <returns>The converted <see cref="Augurk.Entities.ExampleSet"/> instance.</returns>
-        public static ExampleSet ConvertToExampleSet(this SpecFlow.ExampleSet exampleSet)
+        public static ExampleSet ConvertToExampleSet(this Gherkin.Ast.Examples examples)
         {
-            if (exampleSet == null)
+            if (examples == null)
             {
                 throw new ArgumentNullException("exampleSet");
             }
 
             return new ExampleSet()
-                {
-                    Title = exampleSet.Title,
-                    Description = exampleSet.Description,
-                    Keyword = exampleSet.Keyword,
-                    Tags = exampleSet.Tags.ConvertToStrings(),
-                    Columns = exampleSet.Table.Header.Cells.Select(cell => cell.Value).ToArray(),
-                    Rows = exampleSet.Table.Body.Select(row => row.Cells.Select(cell => cell.Value).ToArray()).ToArray()
-                };
-        } 
+            {
+                Title = examples.Name,
+                Description = examples.Description,
+                Keyword = examples.Keyword,
+                Tags = examples.Tags.ConvertToStrings(),
+                Columns = examples.TableHeader.Cells.Select(cell => cell.Value).ToArray(),
+                Rows = examples.TableBody.Select(row => row.Cells.Select(cell => cell.Value).ToArray()).ToArray(),
+                Location = examples.Location.ConvertToSourceLocation()
+            };
+        }
 
         /// <summary>
-        /// Converts the provided <see cref="SpecFlow.ScenarioSteps"/> instance into an enumerable collection of <see cref="Augurk.Entities.Step"/> instances.
+        /// Converts the provided <see cref="Gherkin.Ast.Step"/> instances into an enumerable collection of <see cref="Augurk.Entities.Step"/> instances.
         /// </summary>
-        /// <param name="steps">The <see cref="SpecFlow.ScenarioSteps"/> instance that should be converted.</param>
+        /// <param name="steps">The <see cref="Gherkin.Ast.Step"/> instances that should be converted.</param>
+        /// <param name="dialect">The <see cref="GherkinDialect"/> that is being used for this feature.</param>
         /// <returns>An enumerable collection of <see cref="Augurk.Entities.Step"/> instances.</returns>
-        public static IEnumerable<Step> ConvertToSteps(this SpecFlow.ScenarioSteps steps)
+        public static IEnumerable<Step> ConvertToSteps(this IEnumerable<Gherkin.Ast.Step> steps, GherkinDialect dialect)
         {
             if (steps == null)
             {
                 return new Step[0];
             }
 
-            return steps.Select(step => step.ConvertToStep()).ToArray();
+            var blockKeyword = BlockKeyword.Given;
```

Comment:
Initialize it empty and throw an exception if it's empty when you expected a value. This indicates improper Gherkin.